kubejs/server_scripts/minecraft/recipes.js

```diff
@@ -972,4 +971,24 @@ const registerMinecraftRecipes = (event) => {
         D: '#forge:storage_blocks/coal',
     }).id('tfg:campfire_coal')
 	//#endregion
+
+    //#region Smithing Table
+    event.recipes.tfc.damage_inputs_shaped_crafting(
```

Comment:
you don't need to do the "damage inputs" crafting wrappers when you use gt tools, it handles that already for you